wiki/en/en-Setup.md

```diff
@@ -58,7 +67,10 @@ Once connected to a server, make sure you can hear yourself OK and fix any input
 
 With your sound all set up, you are ready to go. When you connect to a server (you may want to select a genre from the list), the faders you see on the right are your own personal mix. Everything you change here will change what you hear, but won’t affect others. If you move a fader down, that channel will be quieter, if you move it up, the channel will be louder for you.
 
-[screenshot]
+<figure>
+  <img src="{{site.url}}/assets/img/en-screenshots/main-screen-medium.png" loading="lazy" alt="Screenshot" style="width:100%; float:none;">
+  <figcaption>The main window when you are connected to a sever</figcaption>
+</figure>
 
 If you don’t want others to hear your audio, click on the “Mute Myself” button which will stop your audio from being sent to other musicians. They won’t be able to tell you have done this though. But if you see a “mute” icon next to a player, that means they can’t hear you becuase they've muted you in their mix. 
```

Comment:
> if you see a “mute” icon next to a player, that means they can’t hear

Perhaps
```
if you see a “mute” icon next to a channel, that means that person can’t hear
```

Reply:
> becuase 

Typo

Reply:
Ah yes, "player" is not legit by our new style guide. Good spot.

Reply:
> they've muted you

"your channel", I guess, too.